src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1072,7 +1072,7 @@ public EnvVars getEnvironment() {
         buildData.saveBuild(revToBuild);
 
         if (buildData.getBuildsByBranchName().size() >= 100) {
-            log.println("JENKINS-19022: warning: possible memory leak due to Git plugin usage; see: https://wiki.jenkins-ci.org/display/JENKINS/Remove+Git+Plugin+BuildsByBranch+BuildData");
+            log.println("JENKINS-19022: warning: possible memory leak due to Git plugin usage; see: https://wiki.jenkins.io/display/JENKINS/Remove+Git+Plugin+BuildsByBranch+BuildData");
```

Comment:
Perhaps better as a jenkins.io redirect for link longevity.

Reply:
Done in 529928e1e4388a183b461933d8e4f6bfe6ec0b0f .